authmodule/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+spring.security.oauth2.client.registration.google.client-id=
```

Comment:
Let's add little bit of comments here on what is client-id and secret here. and how can a new person can get it.

Reply:
Added some comments related to the same including the link for official documentation on how to generate and use google client-id and secret. Will add more details soon.